lib/rucio/web/ui/static/lifetime_exception.js

```diff
@@ -258,14 +251,15 @@ $(document).ready(function(){
                      comments: reason + '||||' + volume,
                      expires_at: lifetime,
                      success: function(data) {
-                         console.log('New exception created with id ' + data);
                          dialog.dialog( "close");
                          $("#alert_box").html('<div data-alert class="alert-box success radius">The request has been successfully submitted. It will be approved or rejected in the coming days.</div>');
                      },
                      error: function(jqXHR, textStatus, errorThrown) {
-                         console.log('Could not create exception: ' + textStatus);
-                         console.log(errorThrown);
+                         var alert_text = '<div data-alert class="alert-box alert radius"> Problem to create the exception : '
```

Comment:
I cannot approve this. The white space there is just wrong!

Reply:
@cserf push 😄

Reply:
I won't change anything : there is obviously a space before the colon !